plugin.json

```diff
@@ -18,5 +18,10 @@
         "header": "",
         "footer": "",
         "settings": []
+    },
+    "requires_config": {
+        "ServiceSettings": {
```

Comment:
Do we want to include a requirement on the sample? I'd say YES if we're showing how `CheckRequiredConfig` should be used

Reply:
Actually, ideally we'd put the "demo" in our https://github.com/mattermost/mattermost-plugin-demo, and just leave `required_config` as an empty struct here. The intent of the starter plugin is to provide the minimal boilerplate to get started, and adding a demo in this case makes it one more thing to "remove" to actually get started.